src/App.tsx

```diff
@@ -1,18 +1,42 @@
-import React from 'react';
-import './App.css';
+import { FC, useEffect, useState } from 'react';
+import { Pagination } from './components/Pagination';
+
 import { getNumbers } from './utils';
+import './App.css';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42)
   .map(n => `Item ${n}`);
 
-export const App: React.FC = () => {
+export const App: FC = () => {
+  const [currentPage, setCurrentPage] = useState(1);
+  const [perPage, setPerPage] = useState(5);
```

Comment:
5 looks kinda hardcoded here. It is better to create a variable for that to make it clear